Tests/SwiftCurrent_SwiftUITests/SwiftCurrent_SwiftUITests.swift

```diff
@@ -627,4 +627,48 @@ final class SwiftCurrent_SwiftUIConsumerTests: XCTestCase {
 
         wait(for: [expectOnFinish, expectViewLoaded], timeout: TestConstant.timeout)
     }
+
+    func testWorkflowCorrectlyHandlesState() throws {
+        final class Launcher: ObservableObject {
+            var onFinishCalled = false
+            init(workflow: AnyWorkflow,
+                 responder: OrchestrationResponder,
+                 launchArgs: AnyWorkflow.PassedArgs) {
+                if workflow.orchestrationResponder == nil {
+                    workflow.launch(withOrchestrationResponder: responder, passedArgs: launchArgs)
+                }
+            }
+        }
+        struct ModifiedWorkflowViewExplorer<A, W, C>: View, Inspectable {
+            @Binding var isLaunched: Bool
+
+            let inspection = Inspection<Self>()
+            @State var wrapped: W?
+            @State var workflow: AnyWorkflow
+            @State var launchArgs: AnyWorkflow.PassedArgs
+            @State var onFinish = [(AnyWorkflow.PassedArgs) -> Void]()
+            @State var onAbandon = [() -> Void]()
+
+            @StateObject private var model: WorkflowViewModel
```

Comment:
I see these are `private` but the others were made `internal`, but only `wrapped` is accessed?  Any reason for this?

Reply:
In the tests? It really makes no difference. `private` or `public` or `internal` don't change the space they take up in memory. Because this isn't production code it really has no bearing. Any `private` things that are left are just cruft from a copy/paste. However, things that are internal are internal so that you *could* assert on them.